src/components/prompts/FeedbackForm.js

```diff
@@ -235,7 +235,7 @@ class FeedbackPrompt extends React.Component {
             submitting         = { this.state.submitting }
             closeAskPermission = { this.closeAskPermission }
             submit             = { this.submit }
-            question           = { `Is it ok if we save the data you've filled in? We won't save your name or any other information that could give away your identity and it'll help us understand your situation better.` } />
+            question           = { `Is it ok if we save the information you've filled in? We won't save anythign else about you. If you had problems, it'll help us solve them.` } />
```

Comment:
Typo here, "anything" is misspelled.
Not sure we should offer to "solve" their problems. Or at least we could clarify "If you had problems using the app" (as opposed to problems with their life in general).

Reply:
Hehe, thanks for catching those. Hmm... 'help us work on them.'?

Reply:
OK, either way is good for the "help..." part. But I really think it should be "problems using the app" or similar.